components/collector/src/source_collectors/sonarqube/security_warnings.py

```diff
@@ -113,3 +112,11 @@ async def __hotspot_landing_url(self, hotspot_key: str) -> URL:
         component = self._parameter("component")
         branch = self._parameter("branch")
         return URL(f"{url}/security_hotspots?id={component}&branch={branch}&hotspots={hotspot_key}")
+
+    def __issues_with_security_impact_selected(self) -> bool:
+        """Return whether the user selected issues with security impact."""
+        return "issue with security impact" in self._parameter("security_types")
```

Comment:
Why are these fields stringy instead of some stronger type like bool?

Reply:
The parameter can't be bool because there are three options: only issues with security impact, only hotspots, or both.

Reply:
Two bools?